llvm/include/llvm/CodeGen/ValueTypes.h

```diff
@@ -87,6 +87,14 @@ namespace llvm {
       return getExtendedVectorVT(Context, VT, EC);
     }
 
+    /// Returns the EVT that represents a vector tuple type.
+    static EVT getRISCVVectorTupleVT(int Log2LMUL, unsigned NFields) {
```

Comment:
Does this function have any caller?

Reply:
Currently no, let me remove it.

Reply:
Originally it's called in RISCVISelLowering.cpp for lowering seg load/store instructions, but I guess we can use MVT to construct EVT instead.